frontend/src/pages/main-page/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { FC } from 'common/types/types';
+import { Header } from 'components/common/header/header';
+
+import './main-page.scss';
```

Comment:
again. module scss

Reply:
This will be fixed in the next task with customizing the common template

Reply:
@kyrylo-burmelov  This will be fixed in the next task with customizing the common template

Reply:
@artem-drozdov  Fix in this PR please, it takes no more than 30 seconds